src/github/client/github-app-client.ts

```diff
@@ -22,12 +22,13 @@ export class GitHubAppClient extends GitHubClient {
 	private readonly appToken: AuthToken;
 
 	constructor(
+		gitHubConfig: GitHubConfig,
 		logger?: Logger,
-		baseUrl?: string,
+		baseUrl?: string, // goes away once it goes away in the parent class
```

Comment:
goes away because we're using the baseUrl in the config instead right?

Reply:
yep